docs/DeveloperGuide.md

```diff
@@ -508,8 +509,23 @@ testers are expected to do more *exploratory* testing.
  
     2. Test case: `list r/`<br>
        Expected: All archived clients listed. Success message shown. "Archived Client List" is shown below the success message.
-       
+       The color of the priority indicator for each archived client is darkened. 
        ![List All Archived Clients](images/listAllArchivedClients.png)
+       
+### Adding a client
+
+1. Adding a client while the active clients are being shown.
+    
+    1. Prerequisites: List all active clients (see [List Active Clients](#listing-all-active-clients)). Client does not exist. (A client does not exist
+    as long as there is not a client in the clientlist which does not have the same name AND same email or phone number)
+    
+    2. Test case: `add n/John Cena`<br>
+    Expected: The client John Cena is added to the active list. Success message shown. Priority of the user is undefined (white rectangle)
+    
+
+2. Adding a client while the archived clients are being shown.
```

Comment:
Would you pass this part to me to add in the test case? 
Perhaps we can remove this test case, since we didn't include this test case for archive and unarchive commands

Reply:
yeah sure you can add it. The thing is this part is supposed to be jeviers but I added in first.
I feel this should be part of the expected test cases because there were people that did comment about the fact that they have no idea what undefined was

Reply:
ok

Reply:
Approving this first but removing this is pending